### PR TITLE
Improves the main Code Reviews read me to add visibility of contents

### PR DESCRIPTION
**Pull request by magencio**
### What are you trying to address
Improve visibility of the contents in the Code Reviews section of the Playbook.
Issue #496.

### Description of new changes
Adds a detailed summary of contents to the Code Reviews section, including links to all the existing .md files and their different sections.

### CODE REVIEW

File: code-reviews/README.md [hunk 1]
```diff
@@ -18,6 +18,80 @@ Code reviews is a way to have a conversation about the code where participants w
...
 - [Resources](#resources)
 
+### Summary of contents
+
+- [Process Guidance: General](./process-guidance/README.md#general-guidance)
+  - [Pull Requests](./pull-requests.md)
+    - [General Process](./pull-requests.md#general-process)
+    - [Size Guidance](./pull-requests.md#size-guidance)
+    - [Commit Message Standardization](./pull-requests.md#commit-message-standardization)
+    - [Pull Request Template](./pull-request-template.md)
```
Comment (katyamust) [inline code-reviews/README.md:28]: I suggested to move commit message standartization to Source code section. We need to discuss this - I can do it in another PR though

**Review by katyamust — APPROVED** [review]
Very clear now. Thank you!

**Comment by tompaana** [comment]
Am I wrong to say this is effectively a **table of content**? Consider removing "Sections" section and elevate your summary to second level header and rename it to "Table of Contents".

**Comment by magencio** [comment]
@tompaana, agree, so I replaced Sections with Table of Contents

**Review by tompaana — APPROVED** [review]
Weirdly the latest commit doesn't show up when I browse this PR, but I see it in the branch.

But with that 6th commit, I approve. Great change!

**Pull request merged**